docs/markdown/i18n-module.md

```diff
@@ -74,3 +74,70 @@ for normal keywords. In addition it accepts these keywords:
 * `mo_targets` *required*: mo file generation targets as returned by `i18n.gettext()`.
 
 *Added 0.62.0*
+
+
+### i18n.find_xgettext()
+
+``` meson
+i18n.find_xgettext(args: [...], required: true)
+```
+
+Find the `xgettext` program, and return a `XgettextProgram` object.
+`args` are a list of arguments to provide to `xgettext`.
+If `required` is `true` and `xgettext` cannot be found, it will result in
+an error.
+
+This function is to be used when the `gettext` function workflow it not suitable
+for your project. For example, it can be used to produce separate `.pot` files
+for each executable.
+
+*Added 1.7.0*
+
+#### `XgettextProgram` methods
+
+`XgettextProgram` is an [[@external_program]]. Therefore, it has the same methods,
+and can be use in any places requiring an `ExternalProgram`. Furthermore, it has
+the following method:
+
+##### `extract()`
+
+Positional arguments are the following:
+
+- name `str`: the name of the resulting pot file.
+- sources `list[str|File|build_tgt|custom_tgt]`:
+          source files or targets. May be a list of `string`, `File`, [[@build_tgt]],
+          or [[@custom_tgt]] returned from other calls to this function.
+
+Keyword arguments are the following:
+
+- merge `bool`:
+        if `true`, will merge the resulting pot file with extracted pot files
+        related to dependencies of the given source targets. For instance,
+        if you build an executable, then you may want to merge the executable
+        translations with the translations from the dependent libraries.
+- install `bool`: if `true`, will add the resulting pot file to install targets.
+- install_tag `str`: install tag to use for the install target.
+- install_dir `str`: directory where to install the resulting pot file.
+- alias `list[build_tgt]`:
+        a list of build targets using the same resulting pot file. For
+        instance, if you build both a static and a shared library from the
+        same sources, you may specify the static library target as an alias,
+        to use the generated pot file when that static library is used as a
+        dependency of another target.
```

Comment:
I don't really understand what this is doing. But it seems backwards somehow.

Reply:
I removed it. Passing the different build targets as sources would be equivalent, as it deduplicates the source files.